packages/transport/src/api/udp.ts

```diff
@@ -25,25 +25,39 @@ export class UdpApi extends AbstractApi {
                 typeof ERRORS.INTERFACE_DATA_TRANSFER | typeof ERRORS.UNEXPECTED_ERROR
             >
         >(resolve => {
+            signal?.addEventListener('abort', () => {
+                resolve(
+                    this.error({
+                        error: ERRORS.INTERFACE_DATA_TRANSFER,
+                        message: ERRORS.ABORTED_BY_SIGNAL,
+                    }),
```

Comment:
I would rather return only
```
 this.error({
      error: ERRORS.ABORTED_BY_SIGNAL,
 }),
```
we can discuss

Reply:
hmm still thinking about it. its not a big deal. just leaving this note here. maybe it can stay as it is. not a strong opinion.